package.json

```diff
@@ -251,6 +251,10 @@
     }
   },
   "dependencies": {
+    "@radix-ui/colors": "^3.0.0-rc.5",
+    "@radix-ui/react-alert-dialog": "^1.0.5",
```

Comment:
@arthur-lemeur should we keep the radix-colors or should we add our own color scheme ? 

I think we must have a global color css file ? What do you think ?